src/main/groovy/org/gradle/android/AndroidCacheFixPlugin.groovy

```diff
@@ -192,50 +193,40 @@ class AndroidCacheFixPlugin implements Plugin<Project> {
     }
 
     /**
-     * Fix {@link IncrementalTask#getCombinedInput()} relocatability.
+     * Fix {@link IncrementalTask#getCombinedInput()} and {@link StreamBasedTask#getCombinedInput()} relocatability.
      */
-    @FixedInAndroid(version = "3.0.1", link = "https://issuetracker.google.com/issues/68771542#comment3")
+    @AndroidIssue(introducedIn = "3.0.0", fixedIn = "3.0.1", link = "https://issuetracker.google.com/issues/68771542")
     static class IncrementalTask_CombinedInput_Workaround implements Workaround {
```

Comment:
I guess we should change the name to `CombinedInput_Workaround`, WDYT?

Reply:
Alternatively, we could add the same `AndroidIssue` annotation to both workaround classes.

Reply:
Renamed